### PR TITLE
SQL-2102: Publish signing keys for ODBC

### PR DESCRIPTION
**Pull request by nbagnard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,17 +8,24 @@ please see the
 [mongodb/mongo-bi-connector-odbc-driver](https://github.com/mongodb/mongo-bi-connector-odbc-driver)
 repository.
 
+## Verify Integrity of MongoDB ODBC Driver packages
+You can download the [MongoDB ODBC Driver](https://www.mongodb.com/try/download/odbc-driver) from MongoDB Download Center.   
+Select the platform and version you want to verify on that page.   
+Click `Copy link` and use the URL while following the instructions to validate MongoDB packages described [here](https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/).
+
+> :warning: For this [step](!https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/#download-then-import-the-key-file), download and import the MongoDB ODBC Driver public key using this url : `https://pgp.mongodb.com/atlas-sql-odbc.asc`
```
Comment (nbagnard) [inline README.md:16]: This :warning: bit should show up an emoji on Github,
See https://github.com/ikatyang/emoji-cheat-sheet/tree/master#warning

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# mongo-odbc-driver
+# MongoDB ODBC Driver
 
 An 64-bit unicode ODBC driver for connecting to Atlas Data Lake via the MongoSQL dialect.
```
Comment (terakilobyte) [inline README.md:3]:
```suggestion
A 64-bit unicode ODBC driver for connecting to Atlas Data Federation using the MongoSQL dialect.
```

File: README.md [hunk 3]
```diff
@@ -8,17 +8,24 @@ please see the
 [mongodb/mongo-bi-connector-odbc-driver](https://github.com/mongodb/mongo-bi-connector-odbc-driver)
 repository.
 
+## Verify Integrity of MongoDB ODBC Driver packages
```
Comment (terakilobyte) [inline README.md:11]:
```suggestion
## Verify Integrity of MongoDB ODBC Driver Packages
```
Super pedantic, but "Packages" should also be title case here.

File: README.md [hunk 4]
```diff
@@ -8,17 +8,24 @@ please see the
...
+You can download the [MongoDB ODBC Driver](https://www.mongodb.com/try/download/odbc-driver) from MongoDB Download Center.   
+Select the platform and version you want to verify on that page.   
+Click `Copy link` and use the URL while following the instructions to validate MongoDB packages described [here](https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/).
```
Comment (terakilobyte) [inline README.md:14]:
```suggestion
1. Select the platform and version you want to verify on that page.   
2. Click `Copy link` and use the URL while following the instructions to validate MongoDB packages described [here](https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/).
```

File: README.md [hunk 5]
```diff
@@ -8,17 +8,24 @@ please see the
...
 
-## Set up the ODBC driver on Windows
+> :warning:  
+> For this [step](!https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/#download-then-import-the-key-file), download and import the MongoDB ODBC Driver public key using this url : `https://pgp.mongodb.com/atlas-sql-odbc.asc`
+
+## Manually set up the ODBC driver on Windows for local development
```
Comment (terakilobyte) [inline README.md:19]: Just for consistency we may want to choose a style for titles, either Title Case or regular sentence.

**Review by terakilobyte — APPROVED** [review]
LGTM. Changes requested are suggestions. I verified the MSI.

**Review by EthanHardyMongo — APPROVED** [review]
LGTM!

**Pull request merged**